### DISCUSSION TITLE
991AF93F-150F-AAC9-9D91-1B6C3736B51F

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#143] [General]
# 3C 321

3C 321 is a system of two galaxies rotating around each other, notable for showing the first observed galaxy smiting another galaxy with a blast of energy, which is theorized to be from a supermassive black hole at the center of the former galaxy. This characteristic earned it the nickname as the Death Star Galaxy. It is located about 1.2 billion light-years away in the constellation Serpens Caput

https://celestia.mobi/addon?item=991AF93F-150F-AAC9-9D91-1B6C3736B51F&lang=es

**Comment by Pascualito1008** [comment]
Not meteor

**Comment by ahundy66-stack** [comment]
Uhh
